src/pages/messaging/customer-chat-api/index.mdx

```diff
@@ -636,6 +636,7 @@ Returns the dynamic configuration of a given group. It provides data to call [**
 | `organization_id`       | `string`  | The ID of the organization for which the dynamic configuration is returned.                                                          |
 | `client_limit_exceeded` | `bool`    | Specifies if the limit of customers having a chat was exceeded.                                                                      |
 | `domain_allowed`        | `bool`    | Specifies if `url` is configured as a trusted domain (configurable in the Agent Application).                                        |
+| `online_group_ids`      | `array`   | It is returned in response with all the groups' numbers where agents accept chats. It is not visible when there is no such group.    |
```

Comment:
Hello @remleinpiotr, please see the changes connected with the #platform-docs feedback we got. Everything was described in the [Slack thread](https://livechat.slack.com/archives/C2FAU97K2/p1659104442122659). Below, you can find changes in the v3.4 and v3.5

Reply:
Yes, it sounds better, @remleinpiotr WDYT?